frontend/src/components/PrevPointButton.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { makeStyles } from "@material-ui/styles"
+import clsx from "clsx"
+import Button from "@material-ui/core/Button"
+
+const useStyles = makeStyles(() => ({
+  PrevPointButton: {
+    marginTop: 20,
+  },
+}))
+
+const PrevPointButton = props => {
+  // disabled and large are booleans. can add a class where button is imported
+  const classes = useStyles()
+  const { type, children, disabled, large, color, onClick, className } = props
+
+  return (
+    <Button
```

Comment:
I left the same comment I left on @crsmyz for the `<PrevPointInput>` component. if were wrapping components like this then it should spread the props out over the wrapped component in order to maintain compatibility and the override our specific implements details 

`<Input {...props} custom1={value} custom2={value} />`

Reply:
The idea is to actually remove customise-ability  from the Material UI button, at least in terms of styling. The UI for this app is pretty small, and new devs should not be styling a new button, but using the same button across the app.

 If I spread the props, wont that allow all materialUI/Button props to be available again?  is there a better way to limit variability?

Reply:
same thing with the Typography components